### ISSUE TITLE
[Bug]: Incorrectly calls "eventName" on template changes; it should be `appsmith.triggerEvent("onResetClick");` instead.

### ISSUE THREAD

**Issue by phennapa-saeliw** [appsmithorg/appsmith#37395]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

Example code template that is prepared when switching to something else (Vanilla JS, React and Vue) has an incorrect function calling name. It only occurs when the template is changed, but the code that is provided as the default is normal.

### Steps To Reproduce

1. Drag and drop a Custom widget
2. Click on Edit Source button to go to source editor.
3. When switching to another template, it’s found that the code calling incorrect `eventName`, which should be `appsmith.triggerEvent("onResetClick");` not is `appsmith.triggerEvent("onReset");`

### Environment

Production

### Severity

Low (Cosmetic UI issues)

### Issue video log

https://jam.dev/c/13359e04-04bb-4cc3-9131-403652a89ceb

### Version

Cloud - v1.49.0-SNAPSHOT

**Comment by adityarao97** [comment]
Hi @phennapa-saeliw is this issue open? If so, i'd like to contribute to it. Thanks.

**Comment by carinanfonseca** [comment]
Hi @adityarao97, thanks for your interest in contributing! Please feel free to go ahead.

**Comment by ishk99** [comment]
Hi @adityarao97 were you able to fix the issue ?

**Comment by adityarao97** [comment]
@ishk99 No, I was not able to fix this issue, if you want please go ahead.

**Comment by laurenhylan** [comment]
Hi. Is this issue still available? I'd like to contribute .

**Comment by rahulbarwal** [comment]
@laurenhylan Feel free to go ahead and contribute.

**Comment by shubham88fru** [comment]
Hi @rahulbarwal - can you please let me know what is the expected behavior for this issue? I'm asking because a PR was raised for this issue before which seems to be doing the right thing (at least does what the title of the issue says) but didn't get merged.

**Comment by rahulbarwal** [comment]
@shubham88fru  Thanks for bringing our attention to this. We will check the PR and reopen it if everything looks good.

**Comment by AmanAgarwal041** [comment]
This is already fixed by @jsartisan in the commit : https://github.com/appsmithorg/appsmith/pull/38038/files#diff-4668597061a30edd9777210fb9fc1350585f6275ffa1c3775bc35d0993f75faaR67

https://github.com/user-attachments/assets/ad2403fc-9182-42bd-a6a7-f394fb7b2462
